README.md

```diff
@@ -0,0 +1,20 @@
+Aalborg theme
```

Comment:
It's not the aalborg theme, is a Elgg community theme. Please remove any references to the aalborg theme